app/models/lot.rb

```diff
@@ -19,6 +19,9 @@ class Lot < ApplicationRecord
   validates :interesting_categories, presence: { message: 'choose one or more category, or specify a price' },
                                      if: :no_price?
 
+  scope :published, -> { where(status: 'published') }
+  scope :unpublished, -> { where(status: 'unpublished') }
```

Comment:
Not needed. Rails automatically adds such scopes for all enum values: https://api.rubyonrails.org/v5.2.3/classes/ActiveRecord/Enum.html